README.md

```diff
@@ -26,33 +56,173 @@ WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+
+
+# 4. Getting Started
+This section will assist engineers with executing Infrastructure as Code (IaC) found in the _dibbs-aws_ repository utilizing Terraform.  
+
+[Return to Table of Contents](#table-of-contents).
+
+## 4.1 Requirements
+Engineers will need following tools installed on their local machine:
+* Terraform version 1.0.0+  [_See_ Hashicorp installation Guide](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli)
+* AWS CLI version 2+ [_See_ AWS CLI Installation Guide](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-install.html)
+* AWS Profile Access \
+-- <u>Note</u>: Engineers *must* have access and permissions to create AWS resources
+
+[Return to Table of Contents](#table-of-contents).
+
+## 4.2 Clone DIBBS-AWS Repository
+
+<em><strong>4.2.1.</em></strong> Create a directory to store the repository on your local machine
+- Mac OS users: `mkdir workspace`
+- Windows users: `md workspace`
+
+<em><strong>4.2.2.</em></strong> Clone the dibbs-aws repository (<u>use one of the following commands</u>:)\
+&nbsp;&nbsp;&nbsp;&nbsp;`git clone git@github.com:CDCgov/dibbs-cloud.git` <br />  &nbsp;&nbsp;&nbsp;&nbsp;`git clone https://github.com/CDCgov/dibbs-cloud.git`
+
+[Return to Table of Contents](#table-of-contents).
+
+## 4.3 Begin Using Terraform
+
+<em><strong>4.3.1.</em></strong> Navigate to terraform/implementation.\
+&ensp; Initialize your local terraform code. `terraform init`\
+<em><strong>4.3.2.</em></strong> Developing in a terraform workspace.\
+&ensp; Check the terraform workspaces. `terraform workspace list`\
+&ensp; _Note_: If you only have a default terraform workspace, you can create a terraform workspace to develop in.  _Skip to "Create a terraform workspace to develop in," below_.
+- <u>Select the terraform workspace to develop in</u>.\
+ &ensp; `terraform workspace select {selectEnvironmentName}`\
+ &ensp; For example, `terraform workspace select dev`.
+- <u>Create a terraform workspace to develop in</u>.\
+&ensp; `terraform workspace new {newEnvironmentName}`.
+&ensp; For example, `terraform workspace new dev`.
+
+[Return to Table of Contents](#table-of-contents).
+
+## 4.4 Make A New Branch
+Make a new branch to store any of your amendments to ensure you keep a clean main (or master) branch clear from unapproved revisions.
+
+<em><strong>4.4.1.</em></strong> Navigate to the `dibbs-aws` repository on your local machine.\
+&ensp; `cd /workspace/dibbs-aws`\
+<em><strong>4.4.2.</em></strong> Make a new branch.\
+&ensp; For example, `git checkout -b setup-dibbs-aws-backend-and-services`.
+
+[Return to Table of Contents](#table-of-contents).
+
+## 4.5 Update Terraform Through The Command Line
+
+This section will go over some of the sections you will need to amend or change in your local terraform branch.
+
+<em><strong>4.5.1. Update And Setup Your AWS Backend</em></strong>
+* In your terminal, navigate to the _/terraform/setup_ folder (`cd /terraform/setup`).  
+* Run `./setup.sh`.
```

Comment:
I think some of this `/setup` will need to be updated. The current version of this script will attempt to do several things.

- Create a tfvars file
- Create the AWS resources to store the terraform state + Create some local files to help with setting up the ECS infrastructure.
- Update the terraform backend to push the state that was just created

The only time they should have to manually init would be if:

- They want to
- The script fails
- They are upgrading terraform providers

I'd be happy to demo this for the team and to help clear up the docs here.

Reply:
Per our discussion, I made the revisions and will await your review prior to resolving this comment.